pkgs/tools/misc/hexyl/default.nix

```diff
@@ -23,7 +23,7 @@ rustPlatform.buildRustPackage rec {
     '';
     homepage    = https://github.com/sharkdp/hexyl;
     license     = with licenses; [ asl20 /* or */ mit ];
-    maintainers = [];
+    maintainers = with maintainers; [ dywedir ];
     platforms   = platforms.linux ++ platforms.darwin;
```

Comment:
In theory, hexyl 0.3.1 [supports Windows](https://github.com/sharkdp/hexyl/commit/8429b87594386aa020e8cdc421e07d9bc487fe87), though there's no prebuilt binary on the release. I'm not able to test this myself though.